examples/layer2/metal.yml

```
@@ -0,0 +1,121 @@
+---
+- name: Equinix Metal Example Playbook
+  hosts: localhost
+  gather_facts: no
+  tasks:
+    - name: Include the required variables
+      include_vars: "vars/vars.yml"
+
+    # Equinix resources
+    - name: Create a project
+      equinix.cloud.metal_project:
+        name: "{{ project_name }}"
+      register: project
+
+    - name: Enable BGP for the project
+      equinix.cloud.metal_project_bgp_config:
+        project_id: "{{ project.id }}"
+        deployment_type: local
+        asn: 65000
+
+    - name: create a vlan
+      equinix.cloud.metal_vlan:
+        project_id: "{{ project.id }}"
+        metro: "{{ metro }}"
+        vxlan: "1234"
+      register: vlan
+
+    - name: create a VRF
+      equinix.cloud.metal_vrf:
+        name: example-vrf
+        metro: "{{ metro }}"
+        local_asn: "65000"
+        ip_ranges:
```

```suggestion
        local_asn: "{{ metal_vrf_asn }}"
```
Make the Metal VRF ASN a variable so we can use it in the aws playbook too.

---

The existing variable was renamed and referenced here instead of in the virtual circuit configuration